test/common/router/config_impl_test.cc

```diff
@@ -7574,6 +7574,16 @@ TEST_F(ConfigUtilityTest, ParseDirectResponseBody) {
       ConfigUtility::parseDirectResponseBody(route, *api_, MaxResponseBodySizeBytes + 2).size());
 }
 
+TEST_F(ConfigUtilityTest, ParseDirectResponseBodyWithWellKnownEnv) {
+  constexpr uint64_t MaxResponseBodySizeBytes = 4096;
+  envoy::config::route::v3::Route route;
+
+  route.mutable_direct_response()->mutable_body()->set_environment_variable("PATH");
```

Comment:
Consider using an intentional env-var over the well know PATH, e.g.:
```
  TestEnvironment::setEnvVar("ResponseBodyContents", "some body", 1);
  Envoy::Cleanup cleanup([]() { TestEnvironment::unsetEnvVar("ResponseBodyContents"); });
```

Reply:
done, thanks.